server/schedule/filter/metrics.go

```diff
@@ -22,7 +22,7 @@ var (
 			Subsystem: "schedule",
 			Name:      "filter",
 			Help:      "Counter of the filter",
-		}, []string{"action", "address", "store", "type"})
+		}, []string{"action", "address", "store", "acton", "type"})
```

Comment:
I think so.

Reply:
How about `effect`?

Reply:
Good. But I think `affect` is better because `effect` seem to be the result of change.

Reply:
It need to rename `scope` too.